cypress/fixtures/todos/pendingCorrection.json

```diff
@@ -30,7 +30,7 @@
             "isCorrectionPending": false,
             "name": "correction",
             "priority": false,
-            "paymentStatusCode": "COMPLETED",
```

Comment:
A "pending - payment completed" task now displays in the History list, so I simply changed this to "pending - payment incomplete".

Ditto below.